core/project/Dependencies.scala

```diff
@@ -3,13 +3,13 @@ import sbt._
 
 object Dependencies {
 
-  val Scala212 = "2.12.12" // has to be the very same as sbt
-  val Scala213 = "2.13.3" // Scala 2.13.4 breaks scopt when using "--help"
+  val Scala212 = "2.12.15" // has to be the very same as sbt
+  val Scala213 = "2.13.7" // Scala 2.13.4 breaks scopt when using "--help"
```

Comment:
please, note the comment on this line, update `scopt` accordingly and verify that the native-image still works

Reply:
Thanks @andreaTP ;-)

Reply:
```
./kubectl-cloudflow --help
kubectl-cloudflow 2.3.0-RC9-8-c3bf8d62
Usage: kubectl-cloudflow [version|list|status|deploy|undeploy|update-docker-credentials|scale|configure|configuration] [options] <args>...

  -v, --log-level <value>  the logging level
  --kube-config <value>    the kubernetes configuration file
  --help                   prints this usage text
Availble commands:

Command: version [options]
show the current version
  -o, --output <value>     available options are: 'c' or 'classic', 't' or 'table', 'json', 'yaml'
Command: list [options]
list available cloudflow applications
  -n, --namespace <value>  the namespace to be used
  --operator-namespace <value>
                           the namespace where the operator is deployed
  -o, --output <value>     available options are: 'c' or 'classic', 't' or 'table', 'json', 'yaml'
Command: status [options] <cloudflowApp>
show the status of a cloudflow application
  <cloudflowApp>           the name of the cloudflow application
  -n, --namespace <value>  the namespace to be used
  --operator-namespace <value>
                           the namespace where the operator is deployed
  -o, --output <value>     available options are: 'c' or 'classic', 't' or 'table', 'json', 'yaml'
Command: deploy [options] <crFile> [config-key]
deploy a cloudflow applications from a cr file
  --help                   detailed 'deploy' command help
  <crFile>                 the CR file of the cloudflow application
  -u, --username <value>   the docker registry username
  -p, --password <value>   the docker registry password
  --password-stdin         Take the docker registry password from std-in
  --no-registry-credentials
                           No need for registry credentials (e.g. already in the cluster)
  --volume-mount <value>   Key/value pairs of the volume mounts
  --scale <value>          Key/value pairs of streamlets replicas
  --conf <value>           the configuration file/s in HOCON format
  config-key               the configuration keys for the overrides
  --logback-config <value>
                           the logback configuration to be applied
  --unmanaged-runtimes <value>
                           The runtimes that should not be checked
  --serviceaccount <value>
                           the serviceaccount to be used
  -n, --namespace <value>  the namespace to be used
  --operator-namespace <value>
                           the namespace where the operator is deployed
  -o, --output <value>     available options are: 'c' or 'classic', 't' or 'table', 'json', 'yaml'
Command: undeploy [options] <cloudflowApp>
undeploy a cloudflow application
  <cloudflowApp>           the name of the cloudflow application
  -n, --namespace <value>  the namespace to be used
  --operator-namespace <value>
                           the namespace where the operator is deployed
  -o, --output <value>     available options are: 'c' or 'classic', 't' or 'table', 'json', 'yaml'
Command: update-docker-credentials [options] <cloudflowApp> <dockerRegistry>
updates docker registry credentials that are used to pull Cloudflow application images.
  <cloudflowApp>           the name of the cloudflow application
  -n, --namespace <value>  the namespace to be used
  <dockerRegistry>         the name of the docker registry
  -u, --username <value>   the docker registry username
  -p, --password <value>   the docker registry password
  --password-stdin         Take the docker registry password from std-in
  -n, --namespace <value>  the namespace to be used
  --operator-namespace <value>
                           the namespace where the operator is deployed
  -o, --output <value>     available options are: 'c' or 'classic', 't' or 'table', 'json', 'yaml'
Command: scale [options] <cloudflowApp> [<streamlet>=<replicas>]
scales a streamlet of a deployed Cloudflow application to the specified number of replicas
  <cloudflowApp>           the cloudflow application
  <streamlet>=<replicas>   Key/value pairs of streamlets replicas
  -n, --namespace <value>  the namespace to be used
  --operator-namespace <value>
                           the namespace where the operator is deployed
  -o, --output <value>     available options are: 'c' or 'classic', 't' or 'table', 'json', 'yaml'
Command: configure [options] <cloudflowApp> [config-key]
configures a deployed cloudflow application
  --help                   detailed 'configure' command help
  <cloudflowApp>           the cloudflow application
  --conf <value>           the configuration file/s in HOCON format
  config-key               the configuration keys for the overrides
  --logback-config <value>
                           the logback configuration to be applied
  --operator-namespace <value>
                           the namespace where the operator is deployed
  -n, --namespace <value>  the namespace to be used
  --operator-namespace <value>
                           the namespace where the operator is deployed
  -o, --output <value>     available options are: 'c' or 'classic', 't' or 'table', 'json', 'yaml'
Command: configuration [options] <cloudflowApp>
show the current configuration of a cloudflow application
  <cloudflowApp>           the name of the cloudflow application
  -n, --namespace <value>  the namespace to be used
  --operator-namespace <value>
                           the namespace where the operator is deployed
  -o, --output <value>     available options are: 'c' or 'classic', 't' or 'table', 'json', 'yaml'
```
works as expected.